pkg/printers/internalversion/describe.go

```diff
@@ -3273,7 +3273,11 @@ func printNetworkPolicySpecIngressFrom(npirs []networking.NetworkPolicyIngressRu
 		} else {
 			for _, from := range npir.From {
 				w.Write(LEVEL_0, "%s", initialIndent)
-				if from.PodSelector != nil {
+				if from.PodSelector != nil && from.NamespaceSelector != nil {
+					w.Write(LEVEL_0, "%s\n", "From:")
+					w.Write(LEVEL_0, "%s%s%s: %s\n", initialIndent, initialIndent, "NamespaceSelector", metav1.FormatLabelSelector(from.NamespaceSelector))
```

Comment:
The second %s doesn't need to be `initialIndent`, because `initialIndent` is the string before the first char of each line printed in this function. You can use your own indentation, such as two spaces, following the indentation of network policy (i.e. add two more spaces to the next level). 

Other parts of this function use two `initialIndent` as well. You'd want to fix that.